lib/basket.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'menu'
+
+class Basket
+  attr_reader :contents
+
+  def initialize
+    @contents = {}
+  end
+
+  def add_dish(dish, quantity = 1)
```

Comment:
Being able to specify quantity is useful. I meant to implement this but didn't get round to it